src/main/scala/org/allenai/deep_qa/experiments/datasets/Science.scala

```diff
@@ -60,17 +75,55 @@ object ScienceDatasets {
     omnibusMtfGradeEightDevQuestionsWithBuscBackground
   ), "/efs/data/dlfa/processed/omnibus_4_8_dev/")
 
+  /////////////////////////////////////////////////////////////////////
+  // Diagram Multiple True False
+  /////////////////////////////////////////////////////////////////////
+
   val diagramMtfQuestionsWithBuscBackground: JValue = makeBackgroundDataset(
     ScienceFiles.ai2diagramSentences_multipleTrueFalse_appendAnswer,
     "plain sentence",
     ScienceCorpora.buscElasticSearchIndex(10)
   )
 
+  /////////////////////////////////////////////////////////////////////
+  // OpenQA Multiple True False
+  /////////////////////////////////////////////////////////////////////
+
   val openQaMtfQuestionsWithBuscBackground: JValue = makeBackgroundDataset(
     ScienceFiles.openQaAnimalAndGeneralScience_multipleTrueFalse_appendAnswer,
     "plain sentence",
     ScienceCorpora.buscElasticSearchIndex(10))
 
+  /////////////////////////////////////////////////////////////////////
+  // Omnibus QA
+  /////////////////////////////////////////////////////////////////////
+
+  /*
+   * Fourth Grade BUSC Background Files
+   */
+
+  val omnibusQaGradeFourTrainBuscBackgroundFile: JValue = SciQDatasets.makePassageBackgroundFile(
```

Comment:
The `SciQDatasets` reference made me worried there was a bug here for a minute, but you're just using a static reference to some implemented code.  It'd probably be a good idea to have a `DatasetUtil` object, or something similar, that groups all of the `makeFile` and `makeDataset` methods.